packages/patrol/lib/src/native/native_automator.dart

```diff
@@ -447,7 +470,7 @@ class NativeAutomator {
   /// Grants the permission that the currently visible native permission request
   /// dialog is asking for.
   ///
-  /// Does nothing if no permission request dialog is present.
+  /// Throws if no permission request dialog is present.
```

Comment:
Would it be possible to specify the type of possible exceptions, or is it unknown?

Reply:
It will be a `GrpcError`, which I doubt says anything to the tester.

But when some calls fail, we already have error reporting, [here's its impl](https://github.com/leancodepl/patrol/blob/master/packages/patrol/lib/src/native/native_automator.dart#L131-L145) :)